schemas/common/autogeneratedResources.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "https://schema.management.azure.com/schemas/common/autogeneratedResources.json#",
```

Comment:
>id [](start = 3, length = 2)

Do we actually want to call the new schema "autogenerated"? From the customer POV, won't that sound a little weird?